test/unit/test_manifest_download.py

```diff
@@ -0,0 +1,75 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+from mock import patch
+
+from hca.dss import DSSClient
+
+
+def _touch_file(*args, **kwargs):
+    path = args[2] if len(args) > 2 else args[0]
```

Comment:
Explain!

Reply:
Done

Reply:
I see now. Creating a wrapper function for the mock case would be cleared and eliminate the need for an explanation.

Reply:
done.